src/promptflow/promptflow/_cli/data/package_tool/utils.py.jinja2

```diff
@@ -1,9 +1,9 @@
+import yaml
 from pathlib import Path
-import importlib.util
-from promptflow import PFClient
 
 
 package_name = "{{ package_name }}"
+meta_cache_file = Path(__file__).parent.parent / "yamls" / "tool_metas.yaml"
```

Comment:
shall this be in tmp folder?

Reply:
set tool meta cache to %tempfolder%/promptflow/tool_metas/<uuid>.yaml